xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -428,6 +428,12 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem* item,
       case LISTITEM_DIRECTOR:
       case LISTITEM_CHANNEL_NUMBER:
       case LISTITEM_PREMIERED:
+      case VIDEOPLAYER_PARENTAL_RATING:
+      case LISTITEM_PARENTAL_RATING:
+      case LISTITEM_PARENTAL_RATING_CODE:
+      case LISTITEM_MPAA:
```

Comment:
Not sure about MPAA, tbh. Why is it here?

Reply:
As per the description:

`Alias ‘ListItem.ParentalRatingCode’ as ‘ListItem.MPAA’ for compatibility with other skin areas.`

Reply:
Where do you read this?

Reply:
> Where do you read this?

It is in the description of this PR.

Reply:
Ha, you wrote that in the PR description. That does not count as definition. :-) So again, why?

Reply:
Is the data format of ListItem.ParentalRatingCode always compatible with ListItem.MPAA format? I doubt that.

Reply:
> Ha, you wrote that in the PR description. That does not count as definition. :-) So again, why?

To be considerate, empathetic and helpful.  Skinners accustomed the Movie area will be used to referring to this field as 'MPAA'.  Having this alias may be helpful to them.

Reply:
> may be helpful to them.

Remove it, please. If they need it, they will tell us.

Reply:
Done.